src/vtbackend/SixelParser.cpp

```diff
@@ -279,7 +279,7 @@ void SixelParser::leaveState()
                 auto const pad = _params[1];
 
                 auto const imageSize =
-                    _params.size() > 2
+                    _params.size() > 3
```

Comment:
how did you find it? Did this cause a visible bug to the app? If so, we might want to mention that in the changelog. :-)

Reply:
This caused a crush when trying to run [background_select.sh](https://github.com/hackerb9/vt340test/blob/main/j4james/background_select.sh), so not a visible bug

Reply:
nice. thx.